desktop/extensions-sdk/guides/invoke-host-binaries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Invoke host binaries
+description: Add invocations to host binaries from the frontend with the extension SDK.
+keywords: Docker, extensions, sdk, build
+---
+
+In some cases, your extension needs to invoke some command from the host (the computer of your users). For example, you 
+might wand to invoke the CLI of your cloud provider to create a new resource, or the CLI of a tool your extension 
+provides. 
+You could do that executing the CLI from a container with the extension SDK. But this CLI needs to access the host's 
+filesystem, which isn't possible if it runs in a container.
+Host binaries allows exactly this: to invoke binaries from the host. As extensions can run on multiple platforms, 
+this means that you need to ship the binaries for all the platforms you want to support.
+
+In this example, this CLI will be a simple `Hello world` script that must be invoked with a parameter and will return a 
+string. Since the extension will support multiple platforms, the script will be written in `bash` for macOS and Linux, 
+and in `powershell` for Windows.
+
+## Add the binaries to the extension
+Create a file `binaries/unix/hello.sh` with the following content:
+
+```bash
+#!/bin/sh
+echo "Hello, $1!"
+```
+
+Create another file `binaries/windows/hello.ps1` with the following content:
+
+```powershell
+Write-Output "Hello, $args[0]!"
+```
+
+Make them both executable:
+
+```bash
+chmod +x binaries/*
+```
+
+Then update the `Dockerfile` to copy the `binaries` folder into the extension container.
```

Comment:
```suggestion
Then update the `Dockerfile` to copy the `binaries` folder into the extension's image filesystem.
```

Reply:
According to [the docs](https://docs.docker.com/engine/reference/builder/#copy), it is copied to the container's filesystem rather than the image's one. I'll go with `container's filesystem` instead.

`The COPY instruction copies new files or directories from <src> and adds them to the filesystem of the container at the path <dest>.`